services/user.py

```diff
@@ -0,0 +1,56 @@
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist

```

Comment:
The `ObjectDoesNotExist` import is not used in the file. Consider removing it to clean up the code.